example-nextjs/pages/index.tsx

```diff
@@ -45,21 +45,22 @@ eratosthenes(100)`)
     console.log(inputCode)
   }, [inputCode])
 
+  // Note that window.loadPyodide comes from the beforeInteractive pyodide.js Script
+  useEffect(() => {
+    window
+      .loadPyodide({
+        indexURL: 'https://cdn.jsdelivr.net/pyodide/v0.18.1/full/',
+      })
+      .then((pyodide) => setPyodide(pyodide))
+  }, [])
+
   return (
     <div className="max-w-4xl px-4 py-16 mx-auto sm:px-6 lg:px-8">
       <div className="max-w-3xl mx-auto">
-        {/* Content goes here */}
-        <>
-          <Script
-            src="https://cdn.jsdelivr.net/pyodide/v0.18.1/full/pyodide.js"
-            onLoad={async () => {
-              const pyodide = await window.loadPyodide({
-                indexURL: 'https://cdn.jsdelivr.net/pyodide/v0.18.1/full/',
-              })
-              setPyodide(pyodide)
-            }}
-          />
-        </>
+        <Script
+          src="https://cdn.jsdelivr.net/pyodide/v0.18.1/full/pyodide.js"
```

Comment:
Nitpick comment, and not sure if too early for this, but we should consider setting these URLs to be constants somewhere. E.g. `import { PYODIDE_LOADER_URL, PYODIDE_FULL_URL } from './constants;` -- sets a good precedent early on and will be easier to change version numbers, vendors, etc.

Reply:
I'll make a separate issue for this :)